src/test/run-make-fulldeps/alloc-extern-crates/Makefile

```diff
@@ -2,4 +2,4 @@
 
 all:
 	$(RUSTC) fakealloc.rs
-	$(RUSTC) --edition=2018 --crate-type=rlib ../../../../library/alloc/src/lib.rs --cfg feature=\"external_crate\" --extern external=$(TMPDIR)/$(shell $(RUSTC) --print file-names fakealloc.rs)
+	$(RUSTC) --edition=2018 --crate-type=rlib ../../../../library/alloc/src/lib.rs --cfg feature=\"external_crate\" --cfg=feature=\"global-oom-handling\" --extern external=$(TMPDIR)/$(shell $(RUSTC) --print file-names fakealloc.rs)
```

Comment:
This test doesn't seem to test anything anymore: https://github.com/rust-lang/rust/search?q=external_crate

Reply:
By the way I think you can use the same pattern to test liballoc without the `global-oom-handling` feature.

Reply:
Agreed it does seem that way.